src/DotVVM.Framework/Routing/RouteBase.cs

```diff
@@ -153,9 +153,9 @@ public string BuildUrl(IDictionary<string, object> routeValues)
         /// </summary>
         public static void AddOrUpdateParameterCollection(IDictionary<string, object> targetCollection, object anonymousObject)
         {
-            if (anonymousObject is IEnumerable<KeyValuePair<string, string>>)
+            if (anonymousObject is IEnumerable<KeyValuePair<string, object>> pairs)
```

Comment:
This change removes the feature, that `IEnumerable<KeyValuePair<string, string>>` could be used as the parameter object. Could you please add this case back?